card/mod/01_history/set/all/content_history.rb

```diff
@@ -0,0 +1,131 @@
+
+# if these aren't in a nested module, the methods just overwrite the base methods,
+# but we need a distict module so that super will be able to refer to the base methods.
```

Comment:
I don't think I understand this.  There is no base method #content; just methods defined in modules.  In ruby, super doesn't only apply to super classes, it traverses the entire ancestry list, which includes modules.